### ISSUE TITLE
Save to original folder with suffix?

### ISSUE THREAD

**Issue by Stooovie** [Acidham/alfred-image-shrinker#4]
Can we get a "always save to original folder with suffix" feature? Thanks!

**Comment by Acidham** [comment]
added: https://github.com/Acidham/alfred-image-shrinker/releases/tag/v1.5.0

**Comment by Stooovie** [comment]
suffix I see but an option to save to original folder is not there

**Comment by Acidham** [comment]
It saves to original folder with new file plus suffix

**Comment by Stooovie** [comment]
The flow is weird. The workflow requires setting an output folder (or it wouldn't work - `WARNING: Image Shrinker[File Action] Workflow needs to be configured before it can be used.`), and then the shrinked image gets created both in that folder, AND the original folder.

**Comment by Acidham** [comment]
You are right. I released a new version: https://github.com/Acidham/alfred-image-shrinker/releases/latest

Keep the target folder empty to save to source folder.

**Comment by Stooovie** [comment]
Awesome, thank you!
